packages/frontend/src/app/components/header-menu/header-menu.component.html

```diff
@@ -0,0 +1,5 @@
+<ul>
+  <li *ngFor="let tab of tabs; let i = index;">
+      <ag-button [ngClass]="{ 'active': i === activeTabId }" (pressed)="activeTabId = i">{{ tab }}</ag-button>
```

Comment:
As per other components that have been refactored, would be great to avoid any expression templates and instead calculate the computed values in ngOnChanges. It's small detail but, we might as well follow these best practices throughout to set a standard bar.